src/chainparams.cpp

```diff
@@ -61,10 +61,11 @@ class CMainParams : public CChainParams {
         nDefaultPort = 1989;
         nMaxTipAge = 24 * 60 * 60;
         nPruneAfterHeight = 100000;
-        const size_t N = 200, K = 9;
-        BOOST_STATIC_ASSERT(equihash_parameters_acceptable(N, K));
-        nEquihashN = N;
-        nEquihashK = K;
+        eh_epoch_1 = eh200_9;
+        eh_epoch_2 = eh144_5;
+        eh_epoch_1_endblock = 170000;
```

Comment:
If we decide to keep the overlap period, I think `eh_epoch_1_endblock` should be no more than 161152.

Reply:
The more I think about this, the more I would like to see the overlap period only be 10 blocks which would be a 25 minute window.

Reply:
Since the difficulty adjustment is not sensitive to the algorithm, then I think there should be no overlap.

Reply:
I think a little overlap makes sense. It's what we've tested and I would feel more comfortable with a 10 block overlap than one with 0.